core/src/main/java/tech/tablesaw/analytic/AnalyticQuery.java

```diff
@@ -46,60 +46,102 @@ private AnalyticQuery(
   }
 
   /**
-   * Entry point for Full Analytic Query.
+   * Entry point for the fluent analytic query builder. Order By and Partition By are optional.
    *
-   * @return Full Analytic Query Builder.
+   * <p>An AnalyticQuery performs a calculation across a set of table rows that are somehow related
+   * to the current row.
+   *
+   * <pre>
+   * Query includes steps for:
+   * FROM table_name
+   * [ PARTITION BY co1, col2... ]
+   * [ ORDER BY col1, col2...]
+   * window_frame_clause {@link DefineWindowFame}
+   * argument_list {@link AnalyticQuerySteps.AnalyticFunctions}
+   * </pre>
+   *
+   * @return a fluent analytic query builder.
    */
   @Beta
   public static FullAnalyticQuerySteps.FromStep query() {
     return new FullQueryBuilder();
   }
 
   /**
-   * Entry point for Numbering Query.
+   * Entry point for the fluent Numbering Query Builder.
+   *
+   * <p>A numbering assigns integer values to each row based on their position within the specified
+   * partition. Numbering queries require Order By.
```

Comment:
is Numbering different from Rank?  Asking for a friend.

Reply:
Rank is a type of numbering function. This implementation includes the numbering functions row number, rank and dense rank.